code/modules/modular_computers/file_system/programs/radar.dm

```diff
@@ -10,11 +10,11 @@
 	size = 5
 	tgui_id = "NtosRadar"
 	///List of trackable entities. Updated by the scan() proc.
-	var/list/objects
+	var/list/list/objects
```

Comment:
?

Reply:
already addressed this in discord but it's a list of lists and this is fine